.github/workflows/gui.nx.common.yml

```diff
@@ -72,9 +71,10 @@ jobs:
       - name: Build and Push
         uses: elgohr/Publish-Docker-Github-Action@master
         with:
-          name: ${{ steps.get_owner.outputs.owner }}/gui-v2
+          name: ${{ steps.get_owner.outputs.owner }}/common-nx
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+          dockerfile: DockerFile
```

Comment:
I'm not sure if the job will find the dockerfile just with this line or I should add the entire path, ex: `MFE/common/Dockerfile` . @luisfelipefurlan Do you know it?

Reply:
I can also use the `workdir` option but I don't know if is the correct option to use.

Reply:
I saw that the CI of the sidecar micro service uses the `workdir` option, so I believe that the CI is correct.
https://github.com/dojot/dojot/blob/master/.github/workflows/cert-sidecar.yml#L74

Reply:
Leave the workdir with this path and remove the line "dockerfile: DockerFile"
Since the app is using the default Dockerfile name, you can remove the line without any problems.